guides/ansible_engine/3-variables/README.md

```diff
@@ -48,22 +48,21 @@ Add a play definition and some variables to your playbook.  These include addtio
       - mod_wsgi
     apache_test_message: This is a test message
     apache_max_keep_alive_requests: 115
+    apache_webserver_port: 80
 ```
 
 ### Step 3
 
 Add a new task called *httpd packages are present*:
 
 ```yml
-{% raw %}
   tasks:
     - name: Ensure httpd packages are present
       yum:
         name: "{{ item }}"
         state: present
       with_items: "{{ httpd_packages }}"
       notify: restart-apache-service
-{% endraw %}
```

Comment:
Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.